tests/integration/interop-debug-adapter-test.js

```diff
@@ -297,6 +296,25 @@ module('integration/interop-debug-adapter', function(hooks) {
         },
       },
     });
+    await settled();
+
+    this.store.pushPayload(BOOK_MODEL_TYPE, {
+      data: {
+        id: 'urn:bookstore:2',
+        type: BOOK_MODEL_TYPE,
+        attributes: {
+          $type: BOOK_MODEL_TYPE,
+          name: 'The Best Book',
+          author: 'urn:author:2',
+          pubDate: 'Feb 2019',
+          readerComments: ['urn:comment:3', 'urn:comment:4'],
+        },
+      },
+    });
+    await settled();
+
+    this.store.unloadAll(NEW_MODEL_TYPE);
```

Comment:
Why do we unload here?

Reply:
The original test was receiving the unloads triggered by test teardown after the store was destroyed. I converted one to a true update and one to a true remove.